CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 ## 1.6.0 (Unreleased)
+
```

Comment:
Just stumbled [upon this](https://www.terraform.io/docs/extend/community/maintainers.html#versioning-and-changelog):

> To keep the CHANGELOG up to date, we recommend updating the CHANGELOG after every set of commits that change the project. **Never include CHANGELOG updates in a pull request.** If you do, it will very likely cause a merge conflict with other pull requests. Instead, make the pull request without CHANGELOG updates, and add to the CHANGELOG only after merge.

@alexsomesan Should I drop that change from the PR?